code/_globalvars/phobias.dm

```diff
@@ -378,6 +381,20 @@ GLOBAL_LIST_INIT(phobia_objs, list(
 		/obj/structure/spirit_board,
 	)),
 
+	"herecy" = typecacheof(list(
+		/obj/effect/floating_blade,
+		/obj/effect/heretic_influence,
+		/obj/effect/heretic_rune,
+		/obj/effect/rune,
+		/obj/effect/visible_heretic_influence,
+		/obj/item/codex_cicatrix,
+		/obj/item/melee/rune_carver,
+		/obj/item/melee/sickly_blade,
+		/obj/item/toy/eldritch_book,
+		/obj/item/toy/reality_pierce,
+		/obj/structure/destructible/eldritch_crucible,
```

Comment:
Suggestions for other items you could add:
`/obj/item/melee/touch_attack/mansus_fist` will get spooked if you see someone with a spooky hand active
`/obj/item/clothing/mask/madness_mask,` this one is _designed_ to upset people
`/obj/item/reagent_containers/cup/beaker/eldritch,` potion from the crucible
`/obj/item/clothing/suit/hooded/cultrobes/eldritch` if someone's wearing this they're going loud